lib/new_relic/rack/browser_monitoring.rb

```diff
@@ -69,30 +69,33 @@ def should_instrument?(env, status, headers)
 
       private
 
-      def autoinstrument_source(response, headers, js_to_inject)
+      def autoinstrument_source(response, _headers, js_to_inject)
```

Comment:
What do you think about cleaning up the callers and removing the headers arg altogether?

Reply:
This was another case of me sticking to just RuboCop related stuff and not wanting to introduce any other refactoring that was out of scope and potentially bug-ridden. The reason for changing `headers` to `_headers` is that there's actually a cop that will insist on you doing so for unused variables. I'm not sure why it's not on for this project, but I figured I'd bring us in line with it while I was here.

For this specific method, it looks like it only has a single lone caller, so we should be able to do the refactor without too much risk.